redhat-developer/intellij-rsp
# use intellij-commons/ExecHelper (was: ExecUtilClone)

depends on https://github.com/redhat-developer/intellij-common/pull/94 

## What is the purpose of this change? What does it change?
Makes sure intellij-rsp won't break when 2021.3 is released (breaking changes in the API that we use for showing terminals). To achieve this the class `ExecUtilClone` is removed and ExecHelper in intellij-common is used. intellij-common in version 1.2.3-SNAPSHOT with my [changes ](https://github.com/redhat-developer/intellij-common/pull/94)merged is required

## Was the change discussed in an issue?

## How to test changes?
start/stop servers and other tasks that involve opening a terminal. Verify that the terminal is properly opened and the output printed to it.
